airflow/providers/tableau/hooks/tableau.py

```diff
@@ -41,6 +40,16 @@ class TableauHook(BaseHook):
     """
     Connects to the Tableau Server Instance and allows to communicate with it.
 
+    Can be used as a context manager: automatically authenticates the connection
+    when opened and signs out when closed.
+
+    WARNING: authenticaton by personal token was deprecated as Tableau automatically
+    invalidates opened personal token connection if one or more parallel
+    connections with the same token are opened. So, in the environments with
+    multiple parallel tasks this authentication method can lead to numerous bugs
+    and all the jobs will not run as they intended. Therefore personal token
+    auth option is no more available for the end user.
```

Comment:
The place for this is in the docs:
https://github.com/apache/airflow/blob/main/docs/apache-airflow-providers-tableau/connections/tableau.rst